tox.ini

```diff
@@ -6,3 +6,15 @@ commands =
     zope-testrunner --test-path=src []
 deps =
     .[test]
+
+[testenv:coverage]
+commands =
+    coverage run -m zope.testrunner --test-path=src []
+    coverage report -m
+# The combo of skip_install and install_command lets
+# us test the source, meaning coverage reports come out
+# correct
+skip_install = true
+install_command = pip install -e {[testenv]deps} {packages}
```

Comment:
It would be cleaner to use `usedevelop = true` to ask tox to do a `pip install -e .` for you.  Except I don't know if `pip install -e .` followed by `pip install .[test]` would do the right thing, or if it would undo the -e'ness...

The other non-hacky way of having coverage deal with sources living in .tox/py/*lib/site-packages is to add a `[paths]` section to the .coveragerc, like [zope.securitypolicy](https://github.com/zopefoundation/zope.securitypolicy/blob/master/.coveragerc#L8-L12) and several other packages already do.  I think this also requires you to run `coverage combine` before you use `coverage report`?

Anyway, I'm not dead set against this, but I think it is hacky and a bit fragile if somebody changes the deps in [testenv].  I'd rather hardcode the `-e .[test]` bit here.  Also, I wonder if it's good or bad to omit `{options}` from install_command -- I don't remember what pip options tox provides in what circumstances.